### ISSUE TITLE
error seen when c mode is used: error: ValueExtractionSynthesizer could not find:       'cling::runtime::internal::setValueNoAlloc'.

### ISSUE THREAD

**Issue by vickeykumar** [root-project/cling#364]
when c mode is used with cling, below error is seen when trying to print real time values:

$ **cling -xc**

***************** CLING *****************
* Type C code and press enter to run it *
*            Type .q to exit            *
*****************************************
[cling]$ int i
input_line_2:3:2: error: ValueExtractionSynthesizer could not find:
      'cling::runtime::internal::setValueNoAlloc'.
;
 ^
[cling]$ 123
input_line_3:2:2: error: ValueExtractionSynthesizer could not find:
      'cling::runtime::internal::setValueNoAlloc'.
 123
 ^
[cling]$

**Comment by Axel-Naumann** [comment]
Yup we need to disable value printing in C mode, or re-implement it for C.

**Comment by Axel-Naumann** [comment]
https://github.com/root-project/root/pull/9272
